OktaOidc.podspec

```diff
@@ -13,35 +13,18 @@ Integrate your native app with Okta using the AppAuth library.
   s.swift_version = '5.0'
 
   s.subspec 'AppAuth' do |appauth|
-     appauth.source_files = 'Okta/AppAuth/*.{h,m}','Okta/OktaOidc/Internal/OktaUserAgent.{h,m}'
-     appauth.ios.source_files      = 'Okta/AppAuth/iOS/*.{h,m}'
+     appauth.source_files = 'Sources/AppAuth/**/*.{h,m}'
      appauth.ios.deployment_target = '11.0'
-     appauth.osx.source_files = 'Okta/AppAuth/macOS/**/*.{h,m}'
      appauth.osx.deployment_target = '10.10'
   end
 
   s.subspec 'Okta' do |okta|
      okta.dependency 'OktaOidc/AppAuth'
-     okta.subspec 'Classes' do |classes|
-        classes.source_files = 'Okta/OktaOidc/*.{h,m,swift}',
-                                            'Okta/OktaOidc/Internal/*.{h,m,swift}',
-                                            'Okta/OktaOidc/Internal/Tasks/*.{h,m,swift}'
-        classes.exclude_files = 'Okta/OktaOidc/OktaOidc.swift',
-                                              'Okta/OktaOidc/iOS/OktaOidc+Browser.swift',
-                                              'Okta/OktaOidc/macOS/OktaOidc+Browser.swift'
-        classes.ios.source_files = 'Okta/OktaOidc/iOS/*.{swift}',
-                                                  'Okta/OktaOidc/Internal/iOS/*.{swift}',
-                                                  'Okta/OktaOidc/Internal/Tasks/iOS/*.{swift}'
-        classes.ios.deployment_target = '11.0'
-        classes.osx.source_files = 'Okta/OktaOidc/macOS/*.{swift}',
-                                                   'Okta/OktaOidc/Internal/macOS/*.{swift}',
-                                                   'Okta/OktaOidc/Internal/Tasks/macOS/*.{swift}'
-        classes.osx.deployment_target = '10.10'        
-     end
-     okta.source_files = 'Okta/OktaOidc/OktaOidc.swift'
-     okta.ios.source_files = 'Okta/OktaOidc/iOS/OktaOidc+Browser.swift'
-     okta.osx.source_files = 'Okta/OktaOidc/macOS/OktaOidc+Browser.swift'
+     okta.source_files = 'Sources/OktaOidc/**/*.{h,swift}'
```

Comment:
Does it mean that cocoapods installation will include both MacOS and iOS files?

Reply:
It will compile both targets, and the Objective-C headers for both iOS and macOS will be included in the built framework, but the symbols for the different platforms will be conditionally-compiled out when not run on the current target's platform.

This way, there won't be any compatibility differences in the framework based on how it's integrated. I was hoping to avoid situations where the framework behaves differently based on whether it was built using Cocoapods or SPM.